decontamlib/main.py

```diff
@@ -105,9 +105,13 @@ def human_filter_main(argv=None):
     config = get_config(args, args.organism)
 
     fwd_fp = args.forward_reads.name
-    rev_fp = args.reverse_reads.name
     args.forward_reads.close()
-    args.reverse_reads.close()
+    
+    if args.reverse_reads is None:
+        rev_fp = str(None)
```

Comment:
Recommend not converting to string here.  In Python, it is most often best to use None itself to represent a null value.  Later on in the `decontaminate()` method, you can do this:

    if rev_fp is not None:

instead of:

    if rev_fp != str(None):